### ISSUE TITLE
Help: Convert PromiseKit deferred to RxSwift

### ISSUE THREAD

**Issue by pak-amsiq** [ReactiveX/RxSwift#1390]
Hello everyone!

Could someone please help me converting 'promiseKit deferred' to RxSwift:

https://stackoverflow.com/questions/45878470/swift-3-convert-promisekit-deferred-to-rxswift/45879413#45879413

Thank you!

**Comment by sergdort** [comment]
I guess in your use case you can just use 

`rx.sentMessage(#selector(UIViewController.viewDidLayoutSubviews))`

Or use `PublishSubject` 

If you wanna do it only once, just apply `take(1)` operator

Note:
>// Issue: Will be executed on every onCompleted() call

This is very not true, `onCompleted` executed only once as it terminal event

**Comment by sergdort** [comment]
This doesn't look like an issue with the project itself but more of a "how do I do XXX" question. 
Please ask these kinds of questions in our slack channel http://rxswift-slack.herokuapp.com/ or some other more appropriate communication channel.

We are asking this because:
* we can't scale development of this project if people are reporting false issues and we are spending lot of time answering those directly instead of working on this project.
* all who are interested in development of this project and watching it are being spammed by irrelevant information.
* those who want to know what are currently unresolved serious issues are having a harder time figuring it out because list is spammed by non issues, so the real issues are masked.

Thank you for understanding.
